reference/artifacts/in-pipelines/index.md

```diff
@@ -136,6 +136,25 @@ whose execution history to search, and an expected artifact to bind.
 A common use case would be to "promote" the image deployed to staging to a
 pipeline that's deploying to production.
 
+
+## Passing Artifacts Between Pipelines
+
+Artifacts can be passed between pipelines.  If a pipeline triggers the execution of a second pipeline,
+this second pipeline will have access to any artifacts that were available to the first pipeline;
+this includes both artifacts in the first pipeline's trigger as well as artifacts emitted by the
+first pipeline.
+
+Two concrete cases where artifacts can be passed are as follows:
+
+1. __Pipeline triggered by the completion of another pipeline__
```

Comment:
Can you point to the pipeline constructs these two rely on? e.g. "Using a Pipeline trigger" and "Using a Run Pipeline stage". Just to help make it clear how a user could get one pipeline to run another.

Reply:
Done; I also tried to make it a bit clearer by referring to "Pipeline A" and "Pipeline B", but let me know if that makes the working too heavy and hard to read.